src/protocolsupport/protocol/packet/middleimpl/serverbound/play/v_pe/RiderJump.java

```diff
@@ -0,0 +1,35 @@
+package protocolsupport.protocol.packet.middleimpl.serverbound.play.v_pe;
+
+import io.netty.buffer.ByteBuf;
+import protocolsupport.protocol.packet.middle.ServerBoundMiddlePacket;
+import protocolsupport.protocol.packet.middle.serverbound.play.MiddleEntityAction;
+import protocolsupport.protocol.packet.middle.serverbound.play.MiddleSteerVehicle;
+import protocolsupport.protocol.packet.middleimpl.ServerBoundPacketData;
+import protocolsupport.protocol.serializer.VarNumberSerializer;
+import protocolsupport.utils.recyclable.RecyclableArrayList;
+import protocolsupport.utils.recyclable.RecyclableCollection;
+
+public class RiderJump extends ServerBoundMiddlePacket {
+
+	protected long jumptime;
+	int i = 0;
+	
+	@Override
+	public void readFromClientData(ByteBuf clientdata) {
+		jumptime = VarNumberSerializer.readVarLong(clientdata);
+	}
+
+	@Override
+	public RecyclableCollection<ServerBoundPacketData> toNative() {
```

Comment:
comment it out if (And return empty list) it doesn't work.

Reply:
Will do.

Reply:
It seems to work now so I removed the comment.